env.js

```diff
@@ -45,7 +46,14 @@ const shouldValidateEnv =
   (isEASBuild || isLocalBuild) && scriptIsEnvironmentDependant;
 
 const easEnvironmentFileVariable = `ENVIRONMENT_FILE_${APP_ENV.toUpperCase()}`;
-const easEnvironmentFilePath = process.env[easEnvironmentFileVariable];
+
+const environmentFiles = {
+  development: process.env.ENVIRONMENT_FILE_DEVELOPMENT,
+  production: process.env.ENVIRONMENT_FILE_PRODUCTION,
+  staging: process.env.ENVIRONMENT_FILE_STAGING,
+};
+
+const easEnvironmentFilePath = environmentFiles[APP_ENV];
```

Comment:
This had to be change because of an ESLint rule that was throwing an error: [expo/no-dynamic-env-var](https://github.com/expo/expo/blob/main/packages/eslint-plugin-expo/docs/rules/no-dynamic-env-var.md)